nixos/modules/services/networking/bitcoind.nix

```diff
@@ -110,29 +94,36 @@ in {
               bob.passwordHMAC = "b2dd077cb54591a2f3139e69a897ac$4e71f08d48b4347cf8eff3815c0e25ae2e9a4340474079f55705f40574f4ec99";
             }
           '';
-          type = with types; loaOf (submodule rpcUserOpts);
-          description = ''
-            RPC user information for JSON-RPC connnections.
-          '';
+          type = types.attrsOf (types.submodule rpcUserOpts);
+          description = "RPC user information for JSON-RPC connnections.";
         };
       };
 
+      pidFile = mkOption {
```

Comment:
Why does this need to be configurable?

Reply:
It kind of [was there before](https://github.com/NixOS/nixpkgs/blame/55079328c219fb53f3120a34fdba3a80d3be601b/nixos/modules/services/networking/bitcoind.nix#L29), so I didn't want to scratch it completely. I was aiming for as least incompatibility as I could. But you're right and it doesn't need to be that way.